tests/chart_tests/test_airflow_operator.py

```diff
@@ -0,0 +1,103 @@
+from tests import supported_k8s_versions
+from tests.chart_tests.helm_template_generator import render_chart
+import pytest
+
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+class TestAirflowOperator:
+    def test_airflow_operator_cert_manager(self, kube_version):
+        """Test Airflow operator cert manager with flags."""
+        docs = render_chart(
+            validate_objects=False,
+            kube_version=kube_version,
+            values={
+                "airflow-operator": {
+                    "certManager": {"enabled": True},
+                },
+                "global": {
+                    "airflow_operator": {"enabled": True},
```

Comment:
We should be consistent with dashes and underscores if possible. We should choose either `airflow-operator` or `airflow_operator`, not both.

Reply:
it renamed as airflowOperator might be @Shubham-astro should have missed this new  change